src/content/docs/apm/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure.mdx

```diff
@@ -4,12 +4,27 @@ tags:
   - Agents
   - Nodejs agent
   - Hosting services
+  - Azure
+  - Azure Site Extension
 metaDescription: How to install APM for Node.js on Microsoft Azure Web Apps.
 redirects:
   - /docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure
+  - /docs/agents/nodejs-agent/installation-configuration/install-nodejs-agent-azure-site-extension
   - /docs/nodejs/nodejs-agent-on-microsoft-azure
+translate:
+  - jp
+  - kr
 freshnessValidatedDate: never
 ---
+## Installation methods
+
+If you're running a web app and deploying as code on Windows, you can easily install the Node agent using our Azure Site Extension. []See these instructions](/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-azure-site-extension).
+
+If you're deploying a container app, you can follow our []instructions for Docker](/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker).
```

Comment:
```suggestion
If you're deploying a container app, you can follow our [instructions for Docker](/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker).
```

Reply:
container app or containerized app?

Reply:
Container app specifically. The alternative is "containerized web app" or "web app deployed as a container" and they all have different behaviors and constraints.